File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_2/exercice07.py
N1=float(input("saisir un nombre N1: "))
N2=float(input("saisir un nombre N2: "))
N3=float(input("saisir un nombre N3: "))
if (N1<N2 and N2<N3):
    print("les deux nombres sont ordonnées par ordre croissant.")
elif (N1>N2 and N2>N3):
    print("les deux nombres sont ordonnées par ordre décroissant.")
else:
    print("les deux nombres non ordonnés.")

File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_2/exercice07.c
#include <stdio.h>
float N1,N2,N3;
int main(){
    printf("saisir deux nombre N1 et N2: \n");
    scanf("%f %f %f",&N1,&N2,&N3);
    if (N1<N2 && N2<N3)
        printf("les deux nombres sont ordonnées par ordre croissant.\n");
    else if (N1>N2 && N2>N3)
        printf("les deux nombres sont ordonnées par ordre décroissant.\n");
    else
        printf("les deux nombres non ordonnés.\n");
    return 0;   
}